lib/core/spec/lib/jwt_spec.rb

```diff
@@ -3,5 +3,77 @@
 require 'rails_helper'
 
 RSpec.describe Ros::Jwt do
-  pending 'something'
+  subject { described_class.new('Bearer something') }
+
+  context 'when payload is a hash' do
+    context 'when the hash contain value' do
+      subject { described_class.new(some: 'thing', else: 'here') }
```

Comment:
you missed the point that i was making with the `aud`. In my example, passing the `aud` as one of the attributes had a reason for being: I wanted to make sure the code matches the behaviour of JWT. `aud` is part of Ros::JWT defaults, so it should not be settable by passing it via the initializer